sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the implementation of the Azure Client Pipeline</Description>
     <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
-    <Version>1.0.0-preview.5</Version>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Since `Azure.Core` was part of the configuration and storage preview releases, should this also have the `<PreReleaseVersionLabel/>` set to `preview.5` to match the released version?

Reply:
This would be set at queue time. In general we don't want to commit these because we want them to be provided at queue time.